persistent_connection.go

```diff
@@ -228,6 +242,48 @@ func (pconn *persistentConn) hasFeatureWithArg(name, arg string) bool {
 	return found && strings.ToUpper(arg) == val
 }
 
+func (pconn *persistentConn) setClient() error {
+        code, msg, err := pconn.sendCommand("CLNT GoFTP")
+        if err != nil {
+                return err
+        }
+
+        if !positiveCompletionReply(code) {
+                pconn.debug("server doesn't support CLNT: %d-%s", code, msg)
+                return nil
+        }
+
+        return nil
+}
+
+func (pconn *persistentConn) setUnicode() error {
+        code, msg, err := pconn.sendCommand("OPTS UTF8 ON")
```

Comment:
I had an issue with several servers that treat UTF8 in client commands (like path in LIST / MLSD) as ASCII, thus replying with "not found", although they replied with proper UTF-8 paths.